LibSerialize.lua

```diff
@@ -1291,18 +1394,46 @@ function LibSerialize:SerializeEx(opts, ...)
     end
 
     self:_ClearReferences()
+
     return FlushWriter()
 end
 
+function LibSerialize:SerializeAsyncEx(callback, opts, ...)
+    if not CreateFrame then
```

Comment:
Yeah, this isn't sufficient of a deterrent, since the tests still need to run outside the WoW environment.

Reply:
I have an idea here that if the lib is run outside of WoW (or without a callback) then the async function will instead return some simple handlers to operate the coroutine. I'll work on that and update.

Reply:
That won't be sufficient - the async version needs to be able to be tested outside WoW.